contracts/Bitmask.sol

```diff
@@ -0,0 +1,21 @@
+pragma solidity ^0.4.22;
+
+library Bitmask {
+  function checkBit(uint256 bitmask, uint position) internal pure returns (bool) {
```

Comment:
I would call this `hasBit`, and keep the current naming pattern where the `check` prefix is indicative of functions that `require` something. See comments on `checkRole`.

Reply:
that makes sense.  I'll make this change